src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
oh! we're looking at 2 very different mutations (i refactored this one in add-orgs to speed it up).
i could see doom & gloom happening though.

- i create a new user
- i get redirected to the welcome wizard before my new user is created
- i send segment a signal that i got to the welcome wizard

in the above, 2 & 3 are in a race. i fixed that in the add-orgs branch.